### ISSUE TITLE
FATAL: trying to overwrite '/usr/lib/arm-linux-gnueabihf/libnice.so.10

### ISSUE THREAD

**Issue by azoff** [tiny-pilot/ansible-role-ustreamer#75]
Hey team, thanks for developing and maintaining this wonderful library. I'm on a fresh Rasberry PI install and I can't get the installer to get past the `install Janus Debian package` ansible step. Here's the full output:

```
TASK [ansible-role-ustreamer : install Janus Debian package] **********************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "dpkg --force-confdef --force-confold -i /root/.ansible/tmp/ansible-tmp-1668385437.535762-3438-249367551901832/janus_1.0.1-20221104_armhf6028fpmf.deb failed", "stderr": "dpkg: error processing archive /root/.ansible/tmp/ansible-tmp-1668385437.535762-3438-249367551901832/janus_1.0.1-20221104_armhf6028fpmf.deb (--install):\n trying to overwrite '/usr/lib/arm-linux-gnueabihf/libnice.so.10', which is also in package libnice10:armhf 0.1.16-1\nErrors were encountered while processing:\n /root/.ansible/tmp/ansible-tmp-1668385437.535762-3438-249367551901832/janus_1.0.1-20221104_armhf6028fpmf.deb\n", "stderr_lines": ["dpkg: error processing archive /root/.ansible/tmp/ansible-tmp-1668385437.535762-3438-249367551901832/janus_1.0.1-20221104_armhf6028fpmf.deb (--install):", " trying to overwrite '/usr/lib/arm-linux-gnueabihf/libnice.so.10', which is also in package libnice10:armhf 0.1.16-1", "Errors were encountered while processing:", " /root/.ansible/tmp/ansible-tmp-1668385437.535762-3438-249367551901832/janus_1.0.1-20221104_armhf6028fpmf.deb"], "stdout": "(Reading database ... 106286 files and directories currently installed.)\nPreparing to unpack .../janus_1.0.1-20221104_armhf6028fpmf.deb ...\nUnpacking janus (1.0.1) ...\nProcessing triggers for libc-bin (2.31-13+rpt2+rpi1+deb11u4) ...\n", "stdout_lines": ["(Reading database ... 106286 files and directories currently installed.)", "Preparing to unpack .../janus_1.0.1-20221104_armhf6028fpmf.deb ...", "Unpacking janus (1.0.1) ...", "Processing triggers for libc-bin (2.31-13+rpt2+rpi1+deb11u4) ..."]}
```

Any ideas?

**Comment by mtlynch** [comment]
Thanks for reporting this! Can you share the output of this command?

```bash
$ lsb_release -a && uname -a
```

**Comment by azoff** [comment]
```sh
tinypilot@tinypilot:~ $ lsb_release -a && uname -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 11 (bullseye)
Release:	11
Codename:	bullseye
Linux tinypilot 5.15.61-v7l+ #1579 SMP Fri Aug 26 11:13:03 BST 2022 armv7l GNU/Linux
```

**Comment by mtlynch** [comment]
Ah, it looks like Raspbian Bullseye has libnice10 pre-installed, whereas Buster didn't.

A workaround is that you can run `apt remove -y libnice10`, but we'll need a more robust solution here:

https://github.com/tiny-pilot/janus-debian/issues/8

There's no official Janus Debian package for Raspbian, so we had to roll our own, and our inexperience with Debian packaging is kind of biting us.

**Comment by azoff** [comment]
Got it. Ok that works for me. I suppose you can close this or keep it open to track against the issue in question. This is good enough to unblock me.

Thanks again for the prompt attention and support!

**Comment by mtlynch** [comment]
This should now be fixed due to #77
